src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -14,7 +14,9 @@
   <ItemGroup>
     <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\Utils.cs" />
     <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\SerializationTypes.cs" />
+    <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\SerializationTypes.CoreCLR.cs" />
```

Comment:
I see many netcore50 and netcore50aot folders which correspond to platform name and 9 cs files with netcore50aot. None of the file use .NetCore.cs. There are about the same number of files that use CoreCLR. But netcore50aot sound like a specific platform release with version. In our case we only want to differentiate with NetNative.